packages/components/file-uploader-input/stories/file-uploader-input.happo.js

```diff
@@ -1,34 +1,33 @@
-import { storiesOf } from '@open-wc/demoing-storybook';
-import '@tradeshift/elements';
-import '@tradeshift/elements.file-uploader-input';
-
-import { createHappoStories } from '../../../../.storybook-happo/utils';
-
-storiesOf('ts-file-uploader-input', module).add('test', () => {
-	const properties = {
-		rtl: { true: true },
-		disabled: { true: true },
-
-		helpTextMessages: {
-			message: [
-				'Some very very long long help text to give user more information about the input that they need to provide'
-			]
-		},
-		hideFileTypeHelpText: { true: true },
-		size: {
-			full: 'full',
-			medium: 'medium',
-			small: 'small'
-		}
-	};
-
-	const options = {
-		columns: 5,
-		persistent_props: {
-			acceptedFileExtensions: ['doc', 'pdf', 'jpg'],
-			helpTextTitle: 'Help Title'
-		}
-	};
-
-	return createHappoStories('file-uploader-input', properties, '', options);
-});
+// import { storiesOf } from '@open-wc/demoing-storybook';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACommentedCode)